### DISCUSSION TITLE
Circulator informations

### DISCUSSION THREAD

**Discussion by loicguiraud** [dominikamann/oekofen-pellematic-compact#12] [Ideas]
As my new installation gain features, here's the json section for my circulator : 

```
"circ1":{
  "L_pummp":0, 
  "L_ret_temp":243, 
  "L_release_temp":524, 
  "time_prg":0, 
  "mode":1, 
  "pump_release":300, 
  "return_set":550, 
  "name":"Circulateur"
 },

```
It would be great to have those infos

**Comment by dominikamann** [comment]
Hi @loicguiraud 
I will implement at the end of this week. It will take some time (I am quite busy this week). Best Regards

**Comment by dominikamann** [comment]
@loicguiraud are you sure it is "l_pummp" and not "l_pump"?

**Comment by loicguiraud** [comment]
Yes, it’s a copy / paste from the json

**Comment by dominikamann** [comment]
Ok strange. Thanks for the fast reply

**Comment by dominikamann** [comment]
@loicguiraud  I implemented it. Please try it out and let me know if it is working. 

--> https://github.com/dominikamann/oekofen-pellematic-compact/releases/tag/v2.1.0

**Reply by loicguiraud** [reply]
Everything works well, thanks

**Comment by sb89250** [comment]
Hi Guys!

I really appreciate this Home Assistant Integration. Our new Ökofen device is on software 4.02b and they changed "pummp" to "pump". I made this change already in the const.py in my integration and can confirm that with software 4.02b "pump" works.
Maybe you want to program a check if "pummp" or "pump" exists and use the correct value (if the user has an old or new software version).
Unfortunately, I was able to solve this problem on my own, but my skills are only "script-kiddy" and I am not able to help you more than this. The only thing I know is that the software version seems not to be readable in the JSON output.

Kind regards, Sven

**Reply by dominikamann** [reply]
Hi Sven,

thank you I will check and implement it 👍 I will get the new version soon ... then iam able to test it.
